src/material-examples/sort-overview/sort-overview-example.css

```diff
@@ -0,0 +1,3 @@
+.mat-sort-header-container {
```

Comment:
There should be an additional example of the sort directives being applied to an `md-table`.
(we can re-use the same example for both the table and sortable pages)